modules/cnf-modifying-and-applying-the-default-profile.adoc

```diff
@@ -0,0 +1,16 @@
+// CNF-950 4.7 Modifying and applying the default profile
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-modifying-and-applying-the-default-profile_{context}"]
+= Modifying and applying the default profile
+
+You can apply the profile manually or with the toolset of your choice, such as ArgoCD.
+
+This project contains makefile-based tooling that can be used as follows:
```

Comment:
Could it be possible to remove lines 11...16?
Since we have decided not to link the document to the upstream repository, referencing "this project" is not required

Reply:
Removed.